### PR TITLE
[CCXDEV-11023] Add service log tests to PR check

### PR DESCRIPTION
**Pull request by JiriPapousek**
Add relevant IQE tests for template renderer.

### CODE REVIEW

**Review by matysek — APPROVED** [review]

**Review by epapbak — CHANGES_REQUESTED** [review]
I don't see tests running in the pr-check. Is that expected?

File: pr_check.sh [hunk 1]
```diff
@@ -22,14 +22,13 @@ set -exv
 APP_NAME="ccx-data-pipeline"  # name of app-sre "application" folder this component lives in
 COMPONENT_NAME="insights-content-template-renderer"  # name of app-sre "resourceTemplate" in deploy.yaml for this component
 IMAGE="quay.io/cloudservices/insights-content-template-renderer"
-COMPONENTS="insights-content-template-renderer"  # space-separated list of components to laod
+COMPONENTS="ccx-data-pipeline ccx-insights-results dvo-writer ccx-smart-proxy ccx-notification-writer ccx-notification-service ccx-notification-db-cleaner notifications-backend notifications-aggregator notifications-engine insights-content-service ccx-mock-ams ccx-upgrades-sso-mock insights-content-template-renderer"  # space-separated list of components to load
```
Comment (ikerreyes) [inline pr_check.sh:25]: do we need all those for servicelog?

Comment (JiriPapousek) [reply pr_check.sh:25]: I think so, but I'll check once more. It does not seem so, but for example we need aggregator endpoints for the tests themselves and dvo-writer is needed to run the aggregator.

Comment (ikerreyes) [reply pr_check.sh:25]: And do you know why aggregator is needed? For which tests?

**Review by ikerreyes — APPROVED** [review]

**Comment by JiriPapousek** [comment]
> I don't see tests running in the pr-check. Is that expected?

The tests are running now.

**Review by epapbak — APPROVED** [review]

**Pull request merged**